user/troubleshooting/usb-troubleshooting.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: doc
+title: USB Troubleshooting
+permalink: /doc/usb-troubleshooting/
+---
+
+# USB troubleshooting #
+
+## disp-sys-usb does not start ##
+If the disp-sys-usb does not start, it could be due to a PCI passthrough problem. 
+For more details on this issue along with possible solutions, look at [PCI passthrough issues](/doc/pci-troubleshooting/#pci-passthrough-issues).
+
+## Can't attach a USB device / USB device not showing in qvm-usb  ##
+
+Upon trying to attach a USB device using the `qvm-usb -a vm-name device-vm-name:device` command, you may face the error `Device attach failed: no device info received, connection failed, check backend side for details`. 
```

Comment:
This error message is for a different problem than the reset of this section. In case of this message, the issue is specific to the particular device (including things like a broken cable, but also device incompatible with qvm-usb), not sys-usb as a whole.